examples/certified-counter/src/frontend/src/index.ts

```diff
@@ -33,13 +33,13 @@ buttonElement.addEventListener('click', async event => {
 
   const agent = new HttpAgent();
   await agent.fetchRootKey();
-  const tree = await verifyCertification(
-    Principal.fromText(canisterId),
-    new Uint8Array(certificate).buffer,
-    new Uint8Array(witness).buffer,
-    agent.rootKey,
-    50000,
-  );
+  const tree = await verifyCertification({
```

Comment:
Some feedback from David, this would be cleaner as an object rather than separate params. I've no strong opinion either way.